<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CollimatorPrintDialog</class>
 <widget class="QWidget" name="CollimatorPrintDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>918</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: white;
</string>
  </property>
  <widget class="QWidget" name="collimatorWidget" native="true">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-20</y>
     <width>630</width>
     <height>891</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>71</width>
     <height>0</height>
    </size>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>591</width>
      <height>28</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="nameLabel">
       <property name="styleSheet">
        <string notr="true">border-color: black;
border-width: 1px;
border-style: solid;
border-right-style: none;
color: rgb(46, 52, 54);</string>
       </property>
       <property name="text">
        <string>Nom, Pr√©nom:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nameLineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">border-color: black;
border-width: 1px;
border-style: solid;
border-right-style: none;
color: rgb(46, 52, 54);</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="dossierLabel">
       <property name="styleSheet">
        <string notr="true">border-color: black;
border-width: 1px;
border-style: solid;
border-right-style: none;
color: rgb(46, 52, 54);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;N&lt;span style=&quot; vertical-align:super;&quot;&gt;o&lt;/span&gt; de dossier:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="dossierLineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>21</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-color: black;
border-width: 1px;
border-style: solid;
color: rgb(46, 52, 54);</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_87">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>311</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(46, 52, 54);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Collimateur&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="nameLabel_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>37</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(46, 52, 54);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Date:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="dateLineEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>289</width>
      <height>26</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
color: rgb(46, 52, 54);</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>191</width>
      <height>461</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="coll_coordVerticalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout"/>
     </item>
     <item>
      <widget class="QLabel" name="coordinatesLabel">
       <property name="styleSheet">
        <string notr="true">color: rgb(46, 52, 54);</string>
       </property>
       <property name="text">
        <string>Coordinates (x,y)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="coordinatesPlainTextEdit">
       <property name="styleSheet">
        <string notr="true">border: none;
color: rgb(46, 52, 54);
</string>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="CollimatorWidget" name="collimator_drawing" native="true">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>200</y>
      <width>280</width>
      <height>280</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: white;

</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="closePushButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>890</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CollimatorWidget</class>
   <extends>QWidget</extends>
   <header>CollimatorWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>nameLineEdit</tabstop>
  <tabstop>dossierLineEdit</tabstop>
  <tabstop>closePushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
